src/pages/home/Home.js

```diff
@@ -0,0 +1,45 @@
+import React, {useEffect, useState} from 'react';
+import axios from 'axios';
+import './Home.css';
+import {Link} from "react-router-dom";
+
+function Home() {
+    const [posts, setPosts] = useState([]);
+
+    useEffect(() => {
+        async function fetchData() {
+            try {
+                const result = await axios.get('https://www.reddit.com/hot.json?limit=15');
+                console.log(result.data)
+                setPosts(result.data.data.children);
+
+            } catch(e) {
+                console.error(e);
+            }
+        }
+        fetchData(posts);
+    }, []);
+
+
+    return (
+        <main className="main">
+            <h1 className="title">Hottest posts
+                <br/>
+            <span className="span">on Reddit right now</span>
+            </h1>
+            <section className="section">
+            {posts && posts.map((banana) =>
```

Comment:
Als het goed is mag je je check weglaten, omdat je al heel netjes een lege array als default waarde hebt gepakt
```suggestion
            {posts.map((banana) =>
```